src/awx_plugins/interfaces/_temporary_private_container_api.py

```diff
@@ -18,6 +18,8 @@
 def get_incontainer_path(
         path: os.PathLike[str] | str,
         private_data_dir: os.PathLike[str] | str,
+        *,
+        container_root: os.PathLike[str] | str = '',
```

Comment:
@fosterseth would it make sense to just have
```suggestion
        container_root: os.PathLike[str] | str = CONTAINER_ROOT,
```
here instead of that fallback later on?

Reply:
Or maybe have a `None` sentinel instead?
```suggestion
        container_root: os.PathLike[str] | str | None = None,
```

I realized that having an empty string as a magic value for "maybe nothing was passed into the argument" isn't the best look. Semantically, a dedicated sentinel would be better, or a sane default.